ci/travis/install-bazel.sh

```diff
@@ -1,44 +1,78 @@
-set -e
+set -euo pipefail
 
 ROOT_DIR=$(cd "$(dirname "${BASH_SOURCE:-$0}")"; pwd)
 
+version="1.1.0"
+achitecture="${HOSTTYPE}"
 platform="unknown"
-unamestr="$(uname)"
-if [[ "$unamestr" == "Linux" ]]; then
-  echo "Platform is linux."
-  platform="linux"
-elif [[ "$unamestr" == "Darwin" ]]; then
-  echo "Platform is macosx."
-  platform="darwin"
+suffix=".sh"
+case "${OSTYPE}" in
+  msys)
+    echo "Platform is Windows."
+    platform="windows"
+    # No installer for Windows
+    suffix=".exe"
+    ;;
+  darwin*)
+    echo "Platform is Mac OS X."
+    platform="darwin"
+    ;;
+  linux*)
+    echo "Platform is Linux (or WSL)."
+    platform="linux"
+    ;;
+  *)
+    echo "Unrecognized platform."
+    exit 1
+esac
+
+if [ "${OSTYPE}" = "msys" ]; then
+  target="${MINGW_DIR-/usr}/bin/bazel.exe"
+  mkdir -p "${target%/*}"
+  curl -s -L -R -o "${target}" "https://github.com/bazelbuild/bazel/releases/download/${version}/bazel-${version}-${platform}-${achitecture}${suffix}"
 else
-  echo "Unrecognized platform."
-  exit 1
+  target="./install.sh"
+  curl -s -L -R -o "${target}" "https://github.com/bazelbuild/bazel/releases/download/${version}/bazel-${version}-installer-${platform}-${achitecture}${suffix}"
+  chmod +x "${target}"
+  "${target}" --user
+  rm -f "${target}"
 fi
 
-URL="https://github.com/bazelbuild/bazel/releases/download/1.1.0/bazel-1.1.0-installer-${platform}-x86_64.sh"
-wget -O install.sh $URL
-chmod +x install.sh
-./install.sh --user
-rm -f install.sh
-
-if [[ "$TRAVIS" == "true"  ]]; then
+if [ "${TRAVIS-}" = true ]; then
   # Use bazel disk cache if this script is running in Travis.
-  mkdir -p $HOME/ray-bazel-cache
-  echo "build --disk_cache=$HOME/ray-bazel-cache" >> $HOME/.bazelrc
-
-  # Use ray google cloud cache
-  echo "build --remote_cache=https://storage.googleapis.com/ray-bazel-cache" >> $HOME/.bazelrc
-  if [[ "$TRAVIS_PULL_REQUEST" == "false" ]]; then
+  mkdir -p "${HOME}/ray-bazel-cache"
+  echo "build --disk_cache=${HOME}/ray-bazel-cache" >> "${HOME}/.bazelrc"
+fi
+if [ "${TRAVIS-}" = true ] || [ -n "${GITHUB_TOKEN-}" ]; then
+  if [ -n "${encrypted_1c30b31fe1ee_key:+x}" ] && [ -n "${encrypted_1c30b31fe1ee_iv:+x}" ]; then
+    # Use ray google cloud cache
+    echo "build --remote_cache=https://storage.googleapis.com/ray-bazel-cache" >> "${HOME}/.bazelrc"
     # If we are in master build, we can write to the cache as well.
-    openssl aes-256-cbc -K $encrypted_1c30b31fe1ee_key \
-      -iv $encrypted_1c30b31fe1ee_iv \
-      -in $ROOT_DIR/bazel_cache_credential.json.enc \
-      -out $HOME/bazel_cache_credential.json -d
-    echo "build --google_credentials=$HOME/bazel_cache_credential.json" >> $HOME/.bazelrc
-  else
-    echo "build --remote_upload_local_results=false" >> $HOME/.bazelrc
+    upload=0
+    if [ "${TRAVIS_PULL_REQUEST-false}" = false ]; then
+      if [ -n "${BAZEL_CACHE_CREDENTIAL_B64+x}" ]; then
+        {
+          echo "${BAZEL_CACHE_CREDENTIAL_B64}" >> "${HOME}/.bazelrc"
```

Comment:
I think we need to b64 decode `BAZEL_CACHE_CREDENTIAL_B64`

Reply:
Oh whoops I completely forgot! Will fix in a sec.